utils/getScenarioContent.tsx

```diff
@@ -7,51 +7,186 @@
  * description in ScenarioTypeNames for easy(ish) reference.
  */
 
-import { Claim, ClaimDetailsContent, ScenarioContent } from '../types/common'
+import { Claim, ClaimDetailsContent, PendingDetermination, ScenarioContent } from '../types/common'
 import getClaimDetails from './getClaimDetails'
 import getClaimStatus from './getClaimStatus'
+import { isDatePast, isValidDate, parseConvertDate } from './formatDate'
+import { isFirstTimeSlotEarlier } from './timeSlot'
 
 export enum ScenarioType {
   Scenario1,
+  Scenario2,
+  Scenario3,
   Scenario4,
   Scenario5,
   Scenario6,
 }
 
 export const ScenarioTypeNames = {
-  [ScenarioType.Scenario1]: 'Pending determination scenario',
+  [ScenarioType.Scenario1]: 'Determination interview: not yet scheduled',
+  [ScenarioType.Scenario2]: 'Determination interview: scheduled',
+  [ScenarioType.Scenario3]: 'Determination interview: awaiting decision',
   [ScenarioType.Scenario4]: 'Generic pending state: pending weeks',
   [ScenarioType.Scenario5]: 'Base state: no pending weeks, no weeks to certify',
   [ScenarioType.Scenario6]: 'Base state: no pending weeks, weeks to certify',
 }
 
+interface PendingDeterminationScenario {
+  scenarioType: ScenarioType
+  pendingDetermination?: PendingDetermination
+}
+
+export const NonPendingDeterminationValues = ['Canceled', 'Complete', 'TRAN', 'INVL', 'IDNC', '1277', 'OTHR', 'ClmCX']
+
+/**
+ * Determine whether the Determination Status is pending.
+ *
+ * Determination Status is considered pending if:
+ * either DeterminationStatus is blank (NULL)
+ * OR DeterminationStatus is neither "Canceled" nor "Complete" nor "TRAN" nor
+ *   "INVL" nor "IDNC" nor "1277" nor "OTHR" nor "ClmCX"
+ */
+export function isDeterminationStatusPending(pendingDetermination: PendingDetermination): boolean {
+  return (
+    !pendingDetermination.determinationStatus ||
```

Comment:
I _think_ I actually want to be checking for falsy rather than just blank/null. I'm not super confident in the range of values that might come back from the API gateway (see `getScenarioContent.test.tsx` for the full range of values I'm checking for). I assume this might change once we implement tighter validation with #150.